helm-charts/konk/scripts/provision.sh

```diff
@@ -62,6 +62,17 @@ then
   kubectl -n $NAMESPACE label secret $FULLNAME-etcd-ca $LABELS
 fi
 
+if [ "$SCOPE" = "cluster" ]
+then
+  if secret_not_found $FULLNAME-ca $CERT_MANAGER_NAMESPACE
```

Comment:
Was there a technical requirement that the secret lived in cert-manager namespace? Can you document that or link to a jetstack issue describing the limitation?

Reply:
https://docs.cert-manager.io/en/release-0.11/reference/clusterissuers.html
> <p>When referencing a <code class="docutils literal notranslate"><span class="pre">Secret</span></code> resource in <code class="docutils literal notranslate"><span class="pre">ClusterIssuer</span></code> resources (eg <code class="docutils literal notranslate"><span class="pre">apiKeySecretRef</span></code>) the <code class="docutils literal notranslate"><span class="pre">Secret</span></code> needs to be in the same namespace as the <code class="docutils literal notranslate"><span class="pre">cert-manager</span></code> controller pod. You can optionally override this by using the <code class="docutils literal notranslate"><span class="pre">--cluster-resource-namespace</span></code> argument to the controller.</p>